utilities/cli/util.py

```diff
@@ -329,32 +329,78 @@ def get_arch_gpu_str() -> str:
     return arch
 
 
+def _is_valid_sdk_installation(path: Union[str, Path]) -> bool:
+    """
+    Validate if a directory contains a valid Holoscan SDK installation.
+    """
+    path = Path(path) if isinstance(path, str) else path
+    if not path.exists() or not path.is_dir() or not (path / "lib").exists():
+        return False
+    # Check for at least one of these to confirm it's a Holoscan SDK
+    return (path / "include" / "holoscan").exists() or (
+        path / "lib" / "cmake" / "holoscan"
+    ).exists()
```

Comment:
Existence of `holoscan-config.cmake` or `HoloscanConfig.cmake` might be a more concrete check than the `include/holoscan` or `lib/cmake/holoscan` path. See [CMake search strategy](https://cmake.org/cmake/help/v3.26/command/find_package.html#id6) for reference